doc/classes/RID.xml

```diff
@@ -12,10 +12,17 @@
 		<method name="RID">
 			<return type="RID">
 			</return>
-			<argument index="0" name="from" type="Object">
```

Comment:
Missing constructor, but there's none to bound yet. (This likely relied on `Variant.operator RID()`).